IPInfo.IO/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "ipinfo-io"
+version = "0.1.0"
+description = ""
+authors = ["yosra <dirgham.yosra@gmail.com>"]
+readme = "README.md"
+packages = [{include = "ipinfo"}]
+
+[tool.poetry.dependencies]
+python = ">=3.11,<3.12"
+sekoia-automation-sdk = "^1.11.0"
+pytest = "^7.4.4"
+requests-mock = "^1.11.0"
```

Comment:
Test requirements should be moved into a `[tool.poetry.group.dev.dependencies]` section